packages/ui-library/src/components/ui/icon/index.ts

```diff
@@ -8,15 +8,18 @@ import { ClassMapDirective } from 'lit-html/directives/class-map';
 import { until } from 'lit-html/directives/until.js';
 import { unsafeSVG } from 'lit-html/directives/unsafe-svg.js';
 import { TAG_NAME } from './renderFunction';
+import { ThemeType } from '../../../foundation/_tokens-generated/index.themes';
 
 @customElement(TAG_NAME)
 export class BlrIcon extends LitElement {
   static styles = [styleCustom];
 
   @property() icon: IconType = 'blr360Xs';
   @property() size: SizesType = 'md';
-
+  @property() arialabel?: string;
   @property() ignoreSize?: boolean = false;
+  @property() onClick?: () => void;
```

Comment:
not sure if this is up to date.
on develop this event is called blrClick
https://github.com/deven-org/B01LER-Kitchen/blob/develop/packages/ui-library/src/components/ui/icon/index.ts

Reply:
done

Reply:
done